Form/Type/EnumType.php

```diff
@@ -63,23 +61,26 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
         };
 
         $enumClass = function(Options $options) {
-            if ($options->has('data') && is_object($enum = $options->get('data'))) {
+            if ($options->offsetExists('data') && is_object($enum = $options->offsetGet('data'))) {
                 return get_class($enum);
             }
 
             return null;
         };
 
-        $resolver->setDefaults(array(
-            'enum_class' => $enumClass,
-            'choices'    => $choices,
-        ));
-
-        $resolver->setAllowedTypes(array(
-            'enum_class' => 'string'
-        ));
+        $resolver
+            ->setDefined(array('enum_class', 'choices'))
```

Comment:
Strange bug. Tests were run for old (2.1-2.3) versions of Symfony.
https://travis-ci.org/yethee/BiplaneEnumBundle/builds/95997265